src/test/debuginfo/function-prologue-stepping-no-stack-check.rs

```diff
@@ -37,100 +37,79 @@
 // gdb-command:run
 
 // IMMEDIATE ARGS
-// gdb-command:print a
-// gdb-check:$1 = 1
-// gdb-command:print b
-// gdb-check:$2 = true
-// gdb-command:print c
-// gdb-check:$3 = 2.5
+// gdb-command:info args
```

Comment:
Yes, because they are distinct lvalues in MIR. Even if we might skip building allocas for arguments (which would require us using `llvm.dbg.value` instead) and/or elide their moves to variables, we'd still need a notion of "call arguments" (e.g. what shows up in the backtrace) different from pattern bindings.